packages/ui/DatePicker/index.tsx

```diff
@@ -1,18 +1,22 @@
 "use client";
 
-import { useState } from "react";
+import { useContext } from "react";
 
 import DatePicker from "react-datepicker";
 import { DatePickerStyled } from "./DatePicker.styled";
 import "react-datepicker/dist/react-datepicker.css";
+import { LanguageContext } from "../../../apps/web/lib/contexts/LanguageContext";
+
+import { Input } from "../Input";
 
 import en from "date-fns/locale/en-US";
 import pl from "date-fns/locale/pl";
 
 type CustomDatePickerProps = {
-  placeholder: string;
-  lang: string | undefined;
   onSelect: (date: Date) => void;
```

Comment:
changes to fix the clear icon in date picker inputs (2/6)

```suggestion
  onSelect: (date: Date | null) => void;
```